libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -1126,7 +1126,7 @@ string YulUtilFunctions::mappingIndexAccessFunction(MappingType const& _mappingT
 		if (_mappingType.keyType()->isDynamicallySized())
 			return Whiskers(R"(
 				function <functionName>(slot <comma> <key>) -> dataSlot {
-					dataSlot := <hash>(slot <comma> <key>)
+					dataSlot := <hash>(<key> <?+key>,</+key> slot)
```

Comment:
While we're at it, we could use ``<?+key>,</+key>`` in the signature as well and remove ``<comma>`` below...